lang/en/reengagement.php

```diff
@@ -147,3 +151,13 @@
 $string['privacy:metadata:emailtime'] = 'When this user should be emailed';
 $string['privacy:metadata:emailsent'] = 'Email has been sent';
 $string['privacy:metadata:reengagement_inprogress'] = 'Reengagement activities in progress';
+
+$string['course'] = 'Course ID';
+$string['timecreated'] = 'Time Created';
```

Comment:
Moodle uses sentence case, not camel case - so things like "Time Created" should be "Time created" - also "Emailtime" should probably be "Email time"

Reply:
Resolved. Please refer to the new PR for it.